openapi-specs/iot/api/public/alertResolve.yaml

```diff
@@ -37,7 +38,7 @@ requestBody:
           value:
             reason: The alert poses no threat
             resolved: yes
-            reason_type: ['No Action Needed']
+            reason_type: ['Issue Mitigated']
```

Comment:
This appears to be type Array in this example but type `string` in the documentation.

Reply:
> This appears to be type Array in this example but type `string` in the documentation.

great catch! schema updated.